saleor/plugins/tests/test_manager.py

```diff
@@ -875,3 +876,92 @@ def test_list_external_authentications_active_only(channel_USD):
         "id": PluginSample.PLUGIN_ID,
         "name": PluginSample.PLUGIN_NAME,
     } in external_auths
+
+
+def test_run_method_on_plugins_default_value(plugins_manager):
+    default_value = "default"
+    value = plugins_manager._PluginsManager__run_method_on_plugins(
+        method_name="test_method",
+        default_value=default_value,
+    )
+
+    assert value == default_value
+
+
+def test_run_method_on_plugins_default_value_when_not_existing_method_is_called(
+    channel_USD, all_plugins_manager
+):
+    default_value = "default"
+    value = all_plugins_manager._PluginsManager__run_method_on_plugins(
+        method_name="test_method",
+        default_value=default_value,
+    )
+
+    assert value == default_value
+
+
+def test_run_method_on_plugins_value_overridden_by_plugin_method(
+    channel_USD, all_plugins_manager
+):
+    expected = ActiveDummyPaymentGateway.SUPPORTED_CURRENCIES  # last active plugin
+    value = all_plugins_manager._PluginsManager__run_method_on_plugins(
+        method_name="get_supported_currencies",
+        default_value="default_value",
+    )
+
+    assert value == expected
+
+
+@mock.patch(
+    "saleor.plugins.manager.PluginsManager._PluginsManager__run_method_on_single_plugin"
+)
+def test_run_method_on_plugins_only_on_active_ones(
+    mocked_method, channel_USD, all_plugins_manager
+):
+    all_plugins_manager._PluginsManager__run_method_on_plugins(
+        method_name="test_method_name",
+        default_value="default_value",
+    )
+
+    assert len(all_plugins_manager.all_plugins) == 7
+    assert len([p for p in all_plugins_manager.all_plugins if p.active]) == 4
```

Comment:
Would be good to confirm that these 4 plugins are our active plugins, but it can be also in current format if this would require too much effort

Reply:
I could sort out these sample plugins in `sample_plugins.py`  like `inactive_plugins = [...]` etc and use it in fixtures. Would be more readable in tests. WDYT? @korycins

Reply:
I've added checking these ids.